sdk/tvOS/OoyalaTVSkinSDK/OOTVOptionsCollectionViewController.m

```diff
@@ -0,0 +1,142 @@
+//
+//  OOTVOptionsCollectionViewController.m
+//  OoyalaTVSkinSDK
+//
+//  Created by Ileana Padilla on 7/19/17.
+//  Copyright © 2017 ooyala. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "OOTVOptionsCollectionViewController.h"
+#import "OOOoyalaTVConstants.h"
+#import "OOOoyalaTVOptionCell.h"
+#import "OOOoyalaTVPlayerViewController.h"
+#import <OoyalaSDK/OOOoyalaPlayer.h>
+
+// This Pair class is a simple object that holds a string and array.
+@interface Pair : NSObject
+@property (nonatomic, assign) NSString *name;
+@property (nonatomic, assign) NSArray *value;
+@end
+
+@implementation Pair
+@synthesize name, value;
```

Comment:
I don't think you need to synthesize the properties, try if an empty `@implementation` body does the work.